colors.tdesktop-theme

```diff
@@ -365,34 +365,19 @@ profileAdminStartFg: bright_red; // group members list admin star icon
 
 overviewPhotoSelectOverlay: overlay; // shared photos / videos / links fill for selected rows
 
-titleBg: bg0; // window title background when window is inactive
-titleBgActive: titleBg; // window title background when window is active
-titleShadow: windowShadowFg;
-titleButtonBg: none; // custom window title minimize/maximize/restore button background when window is inactive (Windows only)
-titleButtonFg: none; // custom window title minimize/maximize/restore button icon when window is inactive (Windows only)
-titleButtonCloseBg: none; // custom window title close button background when window is inactive (Windows only)
-titleButtonCloseFg: none; // custom window title close button icon when window is inactive (Windows only)
-
-titleButtonBgOver: titleButtonBg; // custom window title minimize/maximize/restore button background with mouse over when window is inactive (Windows only)
-titleButtonFgOver: titleButtonFg; // custom window title minimize/maximize/restore button icon with mouse over when window is inactive (Windows only)
-titleButtonCloseBgOver: titleButtonBgOver; // custom window title close button background with mouse over when window is inactive (Windows only)
-titleButtonCloseFgOver: titleButtonFgOver; // custom window title close button icon with mouse over when window is inactive (Windows only)
-
-titleButtonBgActive: titleButtonBg; // custom window title minimize/maximize/restore button background when window is active (Windows only)
-titleButtonFgActive: titleButtonFg; // custom window title minimize/maximize/restore button icon when window is active (Windows only)
-titleButtonCloseBgActive: titleButtonCloseBg; // custom window title close button background when window is active (Windows only)
-titleButtonCloseFgActive: titleButtonCloseFg; // custom window title close button icon when window is active (Windows only)
-
-titleButtonBgActiveOver: titleButtonBgOver; // custom window title minimize/maximize/restore button background with mouse over when window is active (Windows only)
-titleButtonFgActiveOver: titleButtonFgOver; // custom window title minimize/maximize/restore button icon with mouse over when window is active (Windows only)
-titleButtonCloseBgActiveOver: titleButtonCloseBgOver; // custom window title close button background with mouse over when window is active (Windows only)
```

Comment:
I would prefer that we kept all these specific options, please roll back the remove.